### ISSUE TITLE
Angular: When using ngbTooltip inside OverlayScrollbars the browser hangs

### ISSUE THREAD

**Issue by dinoben** [KingSora/OverlayScrollbars#168]
I was trying to use ngbTooltip inside an OverlayScrollbars element and it caused the browser to hang.
It started working once I changed the overlay-scrollbars element to a regular div.

I was able to replicate the issue on StackBlitz:
https://stackblitz.com/edit/angular-overlayscrollbars-ngbtooltip

Would love your input as to how to solve this :)

**Comment by KingSora** [comment]
Good day!

Well... this issue is related to angulars `zone.js` and `MutationObserver`. Multiple issues were filed, but were closed because of inactivity. [This issue](https://github.com/angular/angular/issues/31712#issuecomment-346521952) is the most recent one. 

Luckily I can provide a workaround for OverlayScrollbars, because for old browsers I'm not using a MutationObserver to observe changes. Simply use this option:

```html
<overlay-scrollbars [options]="{ autoUpdate: true }">
</overlay-scrollbars>
```

**Comment by dinoben** [comment]
> Good day!
> 
> Well... this issue is related to angulars `zone.js` and `MutationObserver`. Multiple issues were filed, but were closed because of inactivity. [This issue](https://github.com/angular/angular/issues/31712) is the most recent one.
> 
> Luckily I can provide a workaround for OverlayScrollbars, because for old browsers I'm not using a MutationObserver to observe changes. Simply use this option:
> 
> ```
> <overlay-scrollbars [options]="{ autoUpdate: true }">
> </overlay-scrollbars>
> ```

Thank you! This worked!

**Comment by KingSora** [comment]
@dinoben no problem! I've found different fix though, it'll be released in the next version :)

**Comment by KingSora** [comment]
The bug is now fixed in the newest release [v1.10.2](https://github.com/KingSora/OverlayScrollbars/releases/tag/v1.10.2) / the newest overlayscrollbars-ngx version [0.2.1](https://www.npmjs.com/package/overlayscrollbars-ngx)
